pkgs/tools/security/john/default.nix

```diff
@@ -131,7 +132,11 @@ stdenv.mkDerivation rec {
 
   meta = with lib; {
     description = "John the Ripper password cracker";
-    license = licenses.gpl2Plus;
+    license =
+      [ licenses.gpl2Plus ]
+      ++
+      # include non-free ClamAV unrar code
```

Comment:
IMO this comment should be moved up to where the flag is defined (so people understand its purpose)

Reply:
OK, done